rust/examples/src/bin/place_order.rs

```diff
@@ -0,0 +1,184 @@
+use std::str::FromStr;
+
+use ellipsis_client::EllipsisClient;
+use phoenix::state::Side;
+use phoenix_sdk::sdk_client::SDKClient;
+use solana_client::nonblocking::rpc_client::RpcClient;
+use solana_sdk::commitment_config::CommitmentConfig;
+#[allow(unused_imports)]
+use solana_sdk::{
+    instruction::Instruction,
+    program_pack::Pack,
+    pubkey::Pubkey,
+    signature::{read_keypair_file, Keypair},
+    signer::Signer,
+};
+use spl_token::state::Mint;
+#[tokio::main]
+async fn main() {
+    // Connect to the solana network and get the market address
+    let network_url = "https://api.devnet.solana.com";
+    let market = Pubkey::from_str("CS2H8nbAVVEUHWPF5extCSymqheQdkd4d7thik6eet9N").unwrap();
+
+    let trader = Keypair::new();
+    let rpc_client =
+        RpcClient::new_with_commitment(network_url.to_string(), CommitmentConfig::confirmed());
+    let client = EllipsisClient::from_rpc(rpc_client, &trader).unwrap();
+    println!("Trader pubkey: {}", trader.pubkey());
+    let sdk = SDKClient::new_from_ellipsis_client_with_all_markets(client)
```

Comment:
since we already know the market, just get single market

Reply:
`new_from_ellipsis_client_with_all_markets` is recommended so I thought it would be good to have in the base example. I added a comment to explain that a SDK can also be created by passing in known market pubkey(s).

Reply:
sg